src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -74919,6 +74919,29 @@ A M()
         return s; 
     }
 } 
+";
+            var comp = CreateCompilation(source);
+            comp.VerifyDiagnostics();
+        }
+
+        [WorkItem(28324, "https://github.com/dotnet/roslyn/issues/28324")]
+        [Fact]
+        public void CopyTypeCustomModifiersWithGenericOverridenMethod()
```

Comment:
>Overriden [](start = 54, length = 9)

Overridden?

Reply:
At some point I should probably appologize for all the typos I've made you see over the years ...